### DISCUSSION TITLE
comment on database missing

### DISCUSSION THREAD

**Discussion by JeroenNienhuis** [dbeaver/dbeaver#20474] [Q&A]
### Description

In my Postgres database there is a comment on the database (created with COMMENT ON DATABASE ... IS ...). In PGadmin 4 I can see the comment, but in BDeaver the description-field is empty
![afbeelding](https://github.com/dbeaver/dbeaver/assets/51078213/36374d6c-74bb-431f-8e26-6405792ac4d1)

I tested it in PostgreSQL 12.11 and 12.15. In an older version, (9.5) it behaves like expected.

### DBeaver Version

Community Version 23.0.5

**Comment by LonwoLonwo** [comment]
Hello @JeroenNienhuis 

Could you please check this query? Does it return a description of your database?

`select description from pg_shdescription join pg_database on objoid = pg_database.oid where datname = 'alg'`

Also, you can find this query in your [Query Manager](https://github.com/dbeaver/dbeaver/wiki/Query-Manager) if you enable all filters.

**Comment by JeroenNienhuis** [comment]
Yes, the query returns a description

The query is visible in the Query Manager, but only as an "SQL / User" type.
On another database from another source the query showed as an "SQL / Util" type returning NULL records and with result "An I/O error occurred while sending to the backend."
Refreshing or closing tabsheet and reopening does not send a new identical sql.
The above test (including alowing all filters) was done without closing DBeaver (program started a few days ago)

Closing DBeaver and restarting then comments do show up.
Some weeks ago I also looked for a comment on the alg-database and could not find any in the GUI.
Could it be some kind of glitch?

**Comment by LonwoLonwo** [comment]
Sorry, I'm confused.
So, did restarting help?
Can you see the description now?

Maybe it was some connection problems. And because of that loading of the database description stopped.

**Comment by JeroenNienhuis** [comment]
Yes, restarting helped, but unknown why. 
For now I'm happy to see the comments.

**Comment by LonwoLonwo** [comment]
Probably, as you said. A connection was closed. Reconnect was after an attempt to read the database description. Did you try to refresh this page? Or close and open again? Usually, it helps in disconnection cases.

**Comment by JeroenNienhuis** [comment]
Usually closing a tabsheet and reopening solves problems with getting data after a loss of connection. 
One of the databases with a comment was opened just yesterday and refreshing didn't help. 
The other database was probably already opened before I lost connection. 
I both cases I did not see any problems because all schemas and tables were visible.

**Comment by LonwoLonwo** [comment]
Description reading is very expensive, and for this reason, this field has a lazy mode. And yes, if a connection is lost, you need to reload this node. I do not see other good options.
